pkg/agent/plugin/workloadattestor/k8s/k8s_test.go

```diff
@@ -554,6 +576,99 @@ func (s *Suite) TestConfigure() {
 	}
 }
 
+func (s *Suite) TestConfigureWithSigstore() {
+	cases := []struct {
+		name    string
+		hcl     string
+		wantErr bool
```

Comment:
can you update this to expect a real error?
so we can verify the error why it is failing is the expected one?